CrashedThresholdWorkerDecoratorV1/fab/CrashedThresholdDecorator/AwareTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
I think from a repo cleanliness standpoint, putting all of these into a single `Decorators/` or `WorkerDecorators/` directory instead of into the root would probably be a good idea.